### PR TITLE
UI: Ux modification

### PR DESCRIPTION
**Pull request by pratap0007**
- Remove download icon
 - Remove Home,Search and Community from Home page
 - Disable upload feature and remove unnecessary files

### CODE REVIEW

**Comment by sthaha** [comment]
@pratap0007 could you please provide link to the deployment?

**Comment by sthaha** [comment]
looks good to me

File: frontend/src/components/main/App.tsx [hunk 1]
```diff
@@ -49,34 +47,43 @@ const App: React.FC<mainProps> = () => {
     onClick: () => console.log('clicked logo'),
     target: '',
   };
-  const logoutUser=()=>{
+  const logoutUser = () => {
```
Comment (PuneetPunamiya) [inline frontend/src/components/main/App.tsx:51]: Is this eslint problem?

Comment (pratap0007) [reply frontend/src/components/main/App.tsx:51]: yeah, space created by eslint .

File: backend/api/cmd/api/main.go [hunk 2]
```diff
@@ -13,7 +13,7 @@ import (
 )
 
 func main() {
-	app, err := app.FromEnv()
+	app, err := app.FromEnv("api")
```
Comment (sthaha) [inline backend/api/cmd/api/main.go:16]: we have the go code back again :(

**Review by sthaha — CHANGES_REQUESTED** [review]
@pratap0007 could you please remove the unrelated files from the commit 🙏 :)

File: backend/api/cmd/api/main.go (hunk 2 above)
Comment (pratap0007) [reply backend/api/cmd/api/main.go:16]: pr on the master branch is updated and here on next branch have to update.

**Pull request closed without merging**